modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -2034,6 +2040,22 @@ It is therefore not supported by the parallel runtime (introduced in Neo4j 5.13)
 For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/planning-and-tuning/runtimes/concepts#runtimes-parallel-runtime[Cypher Manual -> Parallel runtime].
 ====
 
+[[procedure_genai_vector_encodeBatch]]
+.genai.vector.encodeBatch() label:aura-only[Available only on Aura]
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Encode a given batch of resources as vectors using the named provider.
+For each element in the given resource LIST this returns:
```

Comment:
Same questions here: link to GenAI docs? What do we do when releasing for on-prem?

Reply:
We'll remove the label when we release on-prem and add a note in the Cypher manual.